tests/phpunit/tests/theme.php

```diff
@@ -1058,6 +1058,106 @@ static function () {
 		);
 	}
 
+	/**
+	 * Test whether a switched site retrieves the correct stylesheet directory.
+	 *
+	 * @ticket 59677
+	 */
+	public function test_get_stylesheet_directory_with_switched_site() {
+		if ( ! is_multisite() ) {
+			$this->markTestSkipped( 'Test only runs on multisite.' );
+		}
+
```

Comment:
It would be good to add the `ms-required` group so that the tests are skipped prior to running the logic. Looking at the tests, it looks like the `markTestSkipped()` method is no longer used to skip multisite tests.

And per recently encouraged pattern let's add a `@covers` annotation. (cc @costdev).
```suggestion
	 * @ticket 59677
	 * @group ms-required
	 *
	 * @covers ::get_stylesheet_directory
	 */
	public function test_get_stylesheet_directory_with_switched_site() {

Reply:
Oh, nice! That's a good call, thanks!